system/ad_service_state_monitor/config/ad_service_state_monitor.param.yaml

```diff
@@ -83,6 +71,18 @@
           # to be defined for a topic. By default this is set to not use have features.
           # type: ["tier4_perception_msgs/msg/DynamicObjectArray", "tier4_perception_msgs/msg/DynamicObjectWithFeatureArray"]
 
+        /initialpose3d:
+          module: "localization"
+          timeout: 0.0
+          warn_rate: 0.0
+          type: "geometry_msgs/msg/PoseWithCovarianceStamped"
+
+        /localization/pose_twist_fusion_filter/pose:
```

Comment:
`/localization/kinematic_state` is better than `/localization/pose_twist_fusion_filter/pose`.

Reply:
This is just moved to match the order of ad_service_state_monitor.planning_simulation.param.yaml. I plan to remove this node, so I will leave it as it is for now.

Reply:
I see. OK!